docs/README.md

```diff
@@ -0,0 +1,325 @@
+# Design and implementation of Edge Virtualization Engine
+
+1. [Introduction](#introduction)
+2. [Edge Containers](#edge-containers)
+3. [Security](#security)
+4. [EVE Controller](#eve-controller)
+5. [Runtime Configuration](#runtime-configuration)
+6. [Installing EVE on Edge Nodes](#installing-eve-on-edge-nodes)
+7. [Runtime Lifecycle](#runtime-lifecycle)
+8. [Building EVE](#building-eve)
+9. [EVE Internals](#eve-internals)
+
+## Introduction
+
+Edge Virtualization Engine (EVE) is a secure by design operating system
+that supports running *Edge Containers* on compute devices deployed in
+the field. These devices can be IoT gateways, Industrial PCs or general
+purpose ruggedized computers and we will refer to them as *Edge Nodes*.
```

Comment:
"or general purpose ruggedized computers; collectively, these are referred to as *Edge Nodes*."

Reply:
By the way, I like this easing into the terms starting from a place most technologists are comfortable without getting into a formal style. It works well.

Reply:
Thanks @deitch !